src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -108,7 +108,7 @@ def make_comm_msg(
         job_arguments = job_id_like
     elif type(job_id_like) is list:
         job_arguments[JOB_ID_LIST] = job_id_like
-    else:
+    elif job_id_like:
```

Comment:
change this slightly to allow `None` to be passed -- this will create a JobRequest object without any `job_id` or `job_id_list` params